python/dgl/graphbolt/minibatch.py

```diff
@@ -80,6 +80,20 @@ class MiniBatch:
           or other graph components depending on the specific context.
     """
 
+    query_index: Union[torch.Tensor, Dict[str, torch.Tensor]] = None
```

Comment:
```suggestion
    indexes: Union[torch.Tensor, Dict[str, torch.Tensor]] = None
```
Align with torch's definition:
https://lightning.ai/docs/torchmetrics/stable/retrieval/mrr.html